util/src/main/java/io/kubernetes/client/informer/cache/DeltaFIFO.java

```diff
@@ -138,7 +138,7 @@ public void delete(KubernetesObject obj) {
   public void replace(List<KubernetesObject> list, String resourceVersion) {
     lock.writeLock().lock();
     try {
-      Set<String> keys = new HashSet<>();
+      Set<String> keys = new HashSet<>((int) (list.size() / 0.75) + 1);
```

Comment:
thank you for clarifying. while i am not sure if it's an overkilling optimization.. what do you think about moving this to a separate pull so that we can get this one merge?

Reply:
OK,I have reverted this change.
Thank you very much for reviewing.